pkg/agent/metrics/prometheus.go

```diff
@@ -116,6 +116,17 @@ var (
 		[]string{"operation"},
 	)
 
+	OVSMeterPacketCount = metrics.NewGaugeVec(
```

Comment:
The decrease case can happen according to my test. When OVS restarts, the value will be reset to 0.

Reply:
If the only "decrease case" is reset to 0, then a counter is still more appropriate:

> A counter is a cumulative metric that represents a single [monotonically increasing counter](https://en.wikipedia.org/wiki/Monotonic_function) whose value can only increase or be reset to zero on restart.

source: https://prometheus.io/docs/concepts/metric_types/

Reply:
Got it, thank for the details. Will update.

Reply:
Hi Antonin,
It is true that the only "decrease case" I can think of is reset to 0.
I tried to switch to counter, but the problem is that CounterMetric interface only supports inc and add. 
Only the GaugeMetric interface contains set value function.
```
type CounterMetric interface {
	Inc()
	Add(float64)
}
type GaugeMetric interface {
	Set(float64)      ------> The function I want to use.
	Inc()
	Dec()
	Add(float64)
	Write(out *dto.Metric) error
	SetToCurrentTime()
}
```
I would like to get the packet count value and set this value to antrea_agent_ovs_meter_packet_count directly.

Reply:
that makes sense, I think a comment explaining why we chose gauge instead of counter would be appropriate. Something like:

> Even though this metric is monotonically increasing (only being reset to 0 on restart), we define it as a Gauge and not a Counter, as we want to set its value directly using the Set method (using the value provided by OVS), and using Inc / Add is not convenient.

Reply:
Sure, thanks. Will update.